src/App.css

```diff
@@ -0,0 +1,89 @@
+@import url('https://fonts.googleapis.com/css2?family=Nunito+Sans&display=swap');
+
+*{
+    margin: 5px;
+    padding: 0;
+}
+body{
+    background-color: #ffc74a;
+}
+header {
+    height: auto;
+    width: auto;
+    margin-bottom: 30px;
+}
+.logo{
+    display: block;
+    margin-left: auto;
+    margin-right: auto;
+    height:auto;
+    width:25%;
+    position: relative;
```

Comment:
Ik zie geen enkel ander element dat absoluut gepositioneerd staat, dus waar is deze `position: relative` dan voor?